features/change_files_spec.rb

```diff
@@ -16,29 +16,48 @@ class Class1
       end
     RUBY
 
-    is_expected.to match_array(%w[
-                                 ./spec/class1_spec.rb[1:1:1]
-                                 ./spec/class1_spec.rb[1:1:2:1]
-                                 ./spec/class1_spec.rb[1:1:3:1]
-                                 ./spec/class1_spec.rb[1:1:4:1]
-                                 ./spec/class1_spec.rb[1:2:1]
-                                 ./spec/file_spec.rb[1:1]
-                               ])
+    is_expected.to include(
+      './spec/class1_spec.rb[1:1:1]',
+      './spec/class1_spec.rb[1:1:2:1]',
+      './spec/class1_spec.rb[1:1:3:1]',
+      './spec/class1_spec.rb[1:1:4:1]',
+      './spec/class1_spec.rb[1:2:1]',
+      './spec/class1_spec.rb[1:3:1]',
+      './spec/file_spec.rb[1:1]'
+    )
+  end
+
+  it 'generates map if Class1 reopen is changed' do
+    class1_reopen_path.open('w') { |f| f.write <<~RUBY }
+      class Class1
+      end
+    RUBY
+
+    is_expected.to include(
```

Comment:
Why is it `include` now instead of `match_array`? Shouldn't it remain strict?

Reply:
I don't think so. It is not that bad if we predict a bit more

Reply:
Maybe, but these specs allow it to predict a lot more. Is the full output inconveniently longer?